src/parcel.js

```diff
@@ -74,22 +76,19 @@ export default class Parcel extends React.Component {
         return null
       }
     } else {
-      const reactElement = React.createElement(this.props.wrapWith, {ref: this.handleRef})
-
-      if (SingleSpaContext && SingleSpaContext.Consumer) {
-        return (
+      const children = SingleSpaContext && SingleSpaContext.Consumer
+        ? (
           <SingleSpaContext.Consumer>
```

Comment:
This was another bug -- we weren't actually wrapping the parcel in the `wrapWith` element that we were supposed to be doing. So I've changed the code so that we always return a react element that wraps the parcel.